assets/js/googlesitekit/modules/datastore/settings.js

```diff
@@ -123,6 +123,27 @@ export const controls = {
 };
 
 export const selectors = {
+	/**
+	 * Checks whether changes are currently loading for a module.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {string} slug Slug for module store.
+	 * @return {boolean} Whether or not changes are currently loading for the module.
+	 */
+	areSettingsEditDependenciesLoaded: createRegistrySelector(
+		( select ) => ( state, slug ) => {
+			invariant( slug, 'slug is required.' );
+			const storeName = select( CORE_MODULES ).getModuleStoreName( slug );
+			const areSettingsEditDependenciesLoaded =
+				select( storeName )?.areSettingsEditDependenciesLoaded?.();
+			return (
+				areSettingsEditDependenciesLoaded ||
+				areSettingsEditDependenciesLoaded === undefined
+			);
```

Comment:
I don't think the selector should be coerced into a boolean here in all cases. The return here makes sense in that it considers a store without the selector implemented as `true` which is correct, but it doesn't properly handle the case where the module store may not be registered yet, which is very unlikely, but we should return `undefined` early here if `select(storeName)` is empty.